test/export/note_sorter_spec.py

```diff
@@ -0,0 +1,122 @@
+from mamba import describe, it, context, before
+from unittest.mock import MagicMock
+from random import shuffle
+
+from test_utils.anki import mock_anki_modules
+
+mock_anki_modules()
+
+from crowd_anki.export.note_sorter import NoteSorter
+from crowd_anki.config.config_settings import ConfigSettings
+
+
+test_guids = ["aaabbbccc", "bbbcccddd", "cccdddeee", "dddeeefff", "eeefffggg", "fffggghhh"]
```

Comment:
"abc", "bcd" etc would probably be preferable, as the current list makes me start thinking on whether they are the same length, etc, and it's a bit harder to visually compare

Reply:
Sure :+1: